pkg/sdk/streams_impl_gen.go

```diff
@@ -251,12 +251,7 @@ func (r showStreamsDbRow) convert() *Stream {
 		}
 	}
 	if r.BaseTables.Valid {
-		baseTables, err := ParseCommaSeparatedSchemaObjectIdentifierArray(r.BaseTables.String)
-		if err != nil {
-			log.Printf("[DEBUG] error converting show stream: %v", err)
-		} else {
-			s.BaseTables = baseTables
-		}
+		s.BaseTables = ParseCommaSeparatedStringArray(r.BaseTables.String, false)
```

Comment:
Why was it changed, it can return different types of identifiers ?

Reply:
Answered above.